internal/store/migrations.go

```diff
@@ -591,4 +591,29 @@ var migrations = []migration{
 
 		return nil
 	}},
+	{semver.MustParse("0.12.0"), semver.MustParse("0.13.0"), func(e execer) error {
+		// Assign default values to UtilityMetadata if values do not
+		// exist, otherwise, keep existing values
+		result, err := e.Exec(`SELECT * FROM Cluster WHERE UtilityMetadata != ''`)
+		if err != nil {
+			return err
+		}
+
+		numRows, err := result.RowsAffected()
+		if err != nil {
+			return err
+		}
+
+		// if there are UtilityMetadata fields that are not empty, this
+		// table has already been manually migrated
+		if numRows > 0 {
+			return nil
+		}
```

Comment:
This would work if every cluster has been migrated, but I am worried about a scenario where a portion of the clusters have utility metadata, but some don't.

To test this, build two clusters with the schema 11, then use latest master and schema 12 to `provision` one of the clusters. This should work and update the metadata field for that cluster. Then use this branch and update your schema to perform the migration. Check that the provisioned cluster metadata is intact and non-provisioned metadata is set to this migration's value.

Reply:
Yeah okay this does flush out a bug, I see where I had a gap in my testing, and I see how to fix it, so I am doing that now. Good catch, thanks.